firstvoices/backend/tests/test_apis/base_api_test.py

```diff
@@ -285,17 +290,90 @@ def test_detail_403_entry_not_visible(self):
         assert response.status_code == 403
 
 
-class BaseControlledSiteContentApiTest(
-    ListApiTestMixin,
-    ListPermissionsApiTestMixin,
-    DetailApiTestMixin,
-    DetailPermissionsApiTestMixin,
-    BaseSiteContentApiTest,
+class DestroyApiTestMixin:
+    """
+    For use with BaseSiteContentApiTest
+    """
+
+    @pytest.mark.django_db
+    def test_destroy_success_204(self):
+        site, user = factories.get_site_with_member(
+            site_visibility=Visibility.PUBLIC, user_role=Role.LANGUAGE_ADMIN
+        )
+        self.client.force_authenticate(user=user)
+
+        instance = self.create_minimal_instance(site=site, visibility=Visibility.PUBLIC)
+
+        response = self.client.delete(
+            self.get_detail_endpoint(key=instance.id, site_slug=site.slug)
+        )
+
+        assert response.status_code == 204
+        assert response.content == b""  # 0 bytes
+
+    @pytest.mark.django_db
+    def test_destroy_denied_403(self):
+        site = self.create_site_with_non_member(Visibility.PUBLIC)
+        instance = self.create_minimal_instance(site=site, visibility=Visibility.PUBLIC)
+
+        response = self.client.delete(
+            self.get_detail_endpoint(key=instance.id, site_slug=site.slug)
+        )
+
+        assert response.status_code == 403
+
+    @pytest.mark.django_db
+    def test_destroy_missing_404(self):
+        site, user = factories.get_site_with_member(
+            site_visibility=Visibility.PUBLIC, user_role=Role.LANGUAGE_ADMIN
+        )
+        self.client.force_authenticate(user=user)
+
+        response = self.client.delete(
+            self.get_detail_endpoint(key="missing-instance", site_slug=site.slug)
+        )
+
+        assert response.status_code == 404
+
+    @pytest.mark.django_db
+    def test_destroy_site_missing_404(self):
+        site, user = factories.get_site_with_member(
+            site_visibility=Visibility.PUBLIC, user_role=Role.LANGUAGE_ADMIN
+        )
+        self.client.force_authenticate(user=user)
+
+        instance = self.create_minimal_instance(site=site, visibility=Visibility.PUBLIC)
+
+        response = self.client.delete(
+            self.get_detail_endpoint(key=instance.id, site_slug="missing-site")
+        )
+
+        assert response.status_code == 404
+
+
```

Comment:
The base test classes below are reorganized a bit-- I renamed the existing ones to "read-only" and added new ones that include the destroy tests